apps/www/app/content/fundamentals/no/code/cli-config.mdx

```diff
@@ -0,0 +1,173 @@
+  "themes": {
+    "my-theme": {
+      "colors": {
+        "main": { "primary": "#0062BA" },
+        "neutral": "#1E2B3C",
+        "support": { "extra1": "#F45F63" }
+      },
+      "borderRadius": 4,
+      "typography": {
+        "fontFamily": "Inter"
+      }
+    }
+  }
+}
+```
+
+Over har me definert eit tema som heiter `my-theme` med nokre fargar og border radius.
+Legg merke til at `neutral` er definert som ein streng, medan `main` og `support` er objekt. 
+Dette er fordi `neutral` berre har ein farge, medan `main` og `support` har fleire.
+
+## Struktur
+
+| Navn | Type | Påkrevd | Forklaring |
+| ---- | ---- | ------- | ----------- |
+| themes | Objekt | Ja | Inneheld alle tema du vil definere. Ny nøkkel er namnet på temaet. |
+| outDir | Streng | Nei | Mappa der design tokens skal lagrast. Standard er `./design-tokens`. |
+| clean | Boolean | Nei | Slett utdata-mappa før du lagar tokens. Nyttig for å fjerne utdaterte filer. |
+
+### Themes
+
+| Navn | Type | Påkrevd | Forklaring |
+| ---- | ---- | ------- | ----------- |
+| "temaNavn" | Objekt | Ja | Temaet du vil definere. |
+
+Du kan ha fleire tema, og kvart tema kan ha sine eigne fargar, border radius, typografi og overrides.
+
+### Theme
+
+| Navn | Type | Påkrevd | Forklaring |
+| ---- | ---- | ------- | ----------- |
+| colors | Objekt | Ja | Inneheld alle fargar for temaet. |
+| borderRadius | Tal | Ja | Base border radius for temaet. |
+| typography | Objekt | Nei | Inneheld typografi innstillingar for temaet. |
+| overrides | Objekt | Nei | Inneheld overskrivingar for fargar og severity-fargar. |
+
+<Alert data-color="warning">
+Merk at typografi er eksperimentelt. Config skjema kan endre seg når som helst.
+</Alert>
+
+### Colors
+| Navn | Type | Påkrevd | Forklaring |
+| ---- | ---- | ------- | ----------- |
+| main | Objekt | Ja | Hovudfargar for temaet. |
+| neutral | Streng | Ja | Nøytral farge for temaet. |
+| support | Objekt | Nei | Støttefargar for temaet. |
+
+Objektet for main og support inneheld fargar, der nøkkelen er fargen sitt namn og verdien er ein hexkode.
+
+```json
+{
+  "main": {
+    "primary": "#0062BA",
+    "accent": "#1E98F5"
+  },
+  "neutral": "#1E2B3C"
+}
+```
+
+### Overrides
+| Navn | Type | Påkrevd | Forklaring |
+| ---- | ---- | ------- | ----------- |
+| colors | Objekt | Nei | Inneheld overskrivingar for individuelle fargesteg |
+| severityColors | Objekt | Nei | Inneheld overskrivingar for severity-fargar |
+
+#### overrides.colors
+| Navn | Type | Påkrevd | Forklaring |
+| ---- | ---- | ------- | ----------- |
+| "fargeNavn" | Objekt | Nei | Overskrivingar for dominant fargar |
+
+Overstyringen av `colors` lar deg tilpasse spesifikke semantiske fargetokens for kvar farge som er definert i temaets.
+
+Kvar nøkkel i `colors`-objektet skal samsvare med eit fargenavn frå fargane dine.
+
+For kvar farge kan du overstyre individuelle semantiske tokener som `background-default`, `border-subtle`, `text-default` osv.
```

Comment:
perhaps a complete list of all these names so you don't have to go looking for the rest?